tests/clock/ClockUtils.py

```diff
@@ -0,0 +1,384 @@
+import logging
+import random
+import datetime
+import allure
+import time
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.clock.ClockConsts import ClockConsts
+from tests.common.errors import RunAnsibleModuleFail
+
+
+class ClockUtils:
+    @staticmethod
+    def run_cmd(duthosts, cmd, param=''):
+        """
+        @summary:
+            Run a given command and return it's output.
+            * a successful command returns empty output (''), while failure returns error message
+        @return: commands output (str)
+        """
+        with allure.step('Run command: "{}" with param "{}"'.format(cmd, param)):
```

Comment:
this is done throughout the test, I suggest defining a method that will wrap both allure step and log

Reply:
i managed to do it successfully. great idea :-)